docs/fuel-101/transaction-lifecycle.mdx

```diff
@@ -5,4 +5,54 @@ category: Fuel 101
 
 # Transaction Lifecycle
 
-Fuel's transaction lifecycle explained.
+A transaction on the Fuel network goes through a complex series of steps before it is considered final:
+
+1. **Transaction Creation**: Someone sends a transaction to a node.
+2. **Pre-Validation**: The transaction goes through an extensive series of pre-checks, including UTXO validation.
+3. **Transaction Pool**: The transaction gets placed in the transaction pool.
+4. **Selection**: Transactions are selected from the pool.
+5. **Execution & Validation**: The transaction is executed in the Fuel Virtual Machine (Fuel VM).
+6. **Block Production**: The transaction is included in a block.
+7. **Gossiping & Validation**: The block is broadcast to the network and the validatation process is repeated.
+8. **Finality on Ethereum**: The transaction is considered final.
+
+## 1. Transaction Creation
+
+The transaction is created by someone.
+
+One way a transaction can be created is by interacting with an application that uses the `fuels` SDK. In this case, the SDK will do some validation to make sure that the transaction appears valid before sending it to the node. If the SDK detects that the transaction is not valid, the transaction will not move on to the next step.
+
+## 2. Pre-Validation
+
+All transactions are fully [validated](/docs/fuel-101/transaction-model#transaction-validation) by the Fuel VM.
+
+Before a transaction makes it to the transaction pool, it goes through a extensive pre-checks. These checks are done to ensure that the transaction appears valid before attempting to execute it.
+
+## 3. Transaction Pool
+
+The transaction pool sorts pending transactions by their gas price. Note that a transaction's gas represents the total amount of work that the VM did to execute the transaction, while the gas price represents the amount that the transaction is going to pay for each unit of gas (or `GasSpend`).
```

Comment:
We are moving to a dynamic gas price model in testnet, where gas prices are set by the block producer. Users will compete for priority based on 1) FIFO and 2) Tips. 

cc @xgreenx

Reply:
As we're publishing this new doc section asap, we're going to write it for beta-5 and will update accordingly for testnet. let's make sure to add that needed update to the docs sprint plan under the 'testnet launch' category on Notion: https://www.notion.so/fuellabs/Docs-Hub-Sprint-April-2024-e059fb13d3c64568b1464b9616cd1ca8